openapi_python_client/openapi_parser/properties.py

```diff
@@ -15,6 +16,10 @@ class Property:
     constructor_template: ClassVar[Optional[str]] = None
     _type_string: ClassVar[str]
 
+    @property
+    def python_name(self):
```

Comment:
We call `python_name` from a bunch of places now.  Feels inefficient to run the conversion every time.  Could you change this to be a generated attribute in `__post_init__`?

See https://docs.python.org/3/library/dataclasses.html#post-init-processing for details.